docs/content/en/docs/tasks/workload/ingress.md

```diff
@@ -31,12 +19,7 @@ We currently recommend using Emissary-ingress Kubernetes Ingress Controller by A
 
 2. Set up a load balancer: Set up MetalLB Load Balancer by following the instructions [here]({{< relref "./loadbalance" >}})
 
-3. Install Ambassador CRDs and ClusterRoles and RoleBindings
-
-    ```bash
-    kubectl apply -f "https://www.getambassador.io/yaml/ambassador/ambassador-crds.yaml"
-    kubectl apply -f "https://www.getambassador.io/yaml/ambassador/ambassador-rbac.yaml"
-    ```
+3. Install Emissary Ingress: Follow the instructions here [Add Emissary Ingress]({{< relref "../../tasks/packages/emissary" >}})
 
 4. Create Ambassador Service with Type LoadBalancer.
```

Comment:
We don't need this do we?

Reply:
For this one, I think we'd prefer the install of emissary to be done with our controller, rather than via normal kubectl apply.